### DISCUSSION TITLE
.pic.yml debugger not found in vscode

### DISCUSSION THREAD

**Discussion by Jawad102480** [gdsfactory/gdsfactory#2636] [Q&A] [answered]
Hi,
I have the gdsfactory extension installed in my VSCode. But whenever I try to run the mask.pic.yml or pad.pic.yml files, VScode shows a prompt that there is no extension to debug picyml files. I am not being able to find anything from marketplace as well. 

Please suggest a solution.

**Comment by joamatab** [comment, marked as answer]
probably you need VSCode to use your anaconda python

https://gdsfactory.github.io/gdsfactory/notebooks/_3_vscode.html#vscode

**Reply by SadiMJawadAhsan** [reply]
Issue solved. I had to reinstall anaconda and also VSCode
